### PR TITLE
fix: long api references do not load

### PR DESCRIPTION
**Pull request by iliapolo**
An example would be: https://constructs.dev/packages/cdk-iam-floyd/v/0.218.0

This PR puts a hard limit of 1MB on the size of the docs page, anything above that will cause only the README to be displayed, along with a notice that the API reference couldn't be rendered. 

Note that the README displayed is the transliterated README, so at least we have that added value for those packages. 

This is just a stop-gap until we are able to resolve the issue, but at least we won't crash and stall browsers. 

Screen Shot 2021-07-29 at 11 29 55 AM
Screen Shot 2021-07-29 at 12 15 53 PM

### CODE REVIEW

File: src/components/Markdown/Markdown.tsx [hunk 1]
```diff
@@ -115,6 +117,17 @@ export const Markdown: FunctionComponent<{
           return `https://${githubPrefix}/${owner}/${repo}/${githubSuffix}/${url}`;
         };
 
+  const byteLength = Buffer.byteLength(children);
+  if (byteLength > ONE_MEGABYTE) {
+    console.log(`Doc page is larger than 1MB, showing only README...`);
```
Comment (addihorowitz) [inline src/components/Markdown/Markdown.tsx:122]: I guess it's a debug message that we should remove?

**Review by addihorowitz — COMMENTED** [review]
I think there's a debug console message we should remove

File: src/components/Markdown/Markdown.tsx (hunk 1 above)
Comment (iliapolo) [reply src/components/Markdown/Markdown.tsx]: Removed it.

File: src/components/Markdown/Markdown.tsx [hunk 2]
```diff
@@ -13,6 +13,8 @@ import { Ul, Ol, Li } from "./List";
 import { Table, Thead, Tbody, Tfoot, Tr, Th, Td, TableCaption } from "./Table";
 import { A, Blockquote, Em, P, Pre, Sup } from "./Text";
 
+const ONE_MEGABYTE = 1024 * 1024;
```
Comment (iliapolo) [inline src/components/Markdown/Markdown.tsx:16]: @gabewomble do you think thats a reasonable limit? can we make it bigger?

**Review by addihorowitz — APPROVED** [review]

**Pull request merged**